<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OpenZIP</class>
 <widget class="QMainWindow" name="OpenZIP">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OpenZIP</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QListView" name="fileViewer">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>641</width>
      <height>421</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="extract">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>430</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract</string>
    </property>
   </widget>
   <widget class="QPushButton" name="extractSelected">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>430</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Extract selected file</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>640</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpen"/>
    <addaction name="actionew"/>
    <addaction name="separator"/>
    <addaction name="actionExir"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
    <addaction name="actionAbout_OpenZIP"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuAbout"/>
  </widget>
  <action name="actionOpen">
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionew">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionExir">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionAbout_OpenZIP">
   <property name="text">
    <string>About OpenZIP</string>
   </property>
  </action>
  <action name="actionUpdate_from_UPD_file">
   <property name="text">
    <string>Update from UPD file</string>
   </property>
  </action>
  <action name="actionUpdate_from_Github">
   <property name="text">
    <string>Update from Github</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
